operators/cuda/add_mul.h

```diff
@@ -190,7 +190,7 @@ struct AddAndMul {
 *  A[1, J] + B[I, J] is supported,
 *  A[1, J, 1] + B[I, J, K] is not supported,
 *
-* If negative is true, then the output is transposed, then
```

Comment:
Does this op support `subtract_first=false` and `negative=true` and all other combinations?

Reply:
Yes, I chose the following convention: a parameter is a template parameter if it changes the kernel name. So substract_first is not seen by the user but it implicitly set it up by choosing op SubMul or MulSub. A parameter is an argument of the kernel if the name remains the same.